src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -61,9 +61,13 @@ export const _UserAvatar: React.FC<UserAvatarProps> = ({
   } = useVenueUserStatuses(user);
 
   const avatarSrc = useMemo((): string => {
+    const validAvatar =
+      (user?.pictureUrl?.includes("static") ||
+        user?.pictureUrl?.includes("http")) &&
+      user?.pictureUrl;
```

Comment:
```suggestion
    // @debt extract utility functions for proper checks of http and static using RegExp's
    const validAvatar =
      (user?.pictureUrl?.includes("static") ||
        user?.pictureUrl?.includes("http")) &&
      user?.pictureUrl;
```


[NOTE]
These should probably be extracted into util functions and refactored with regexp that filters out edge cases like `http` being in the middle of the string.

Reply:
replaced `includes("http")` with `startsWith("http")`, but more to be done in the future